docs/content/concepts/authentication.md

```diff
@@ -62,6 +62,229 @@ The activation for the individual tenant:
+nocheckcert = false;
+cafile = "[CAFile]";
+sslkey = "/etc/ssl/certs/[ID].key";
+sslcert = "/etc/ssl/certs/[ID].crt";
+};
+```
+
+If your client service is a linux, you can use the command bellow to see the logs produced by the swupdate.
+
+`journalctl --follow -u swupdate`
+
+#### Hawkbit Configurations
+
+There are also some configurations that you need update when you are deploying your hawkbit service. If your deploying it as a docker container you need to update your docker-compose.yml file such as,
+
+```yml
+version: '3'
+
+services:
+  # HawkBit service
+  hawkbit:
+    image: "hawkbit/hawkbit-update-server:latest-mysql"
+    environment:
+      SPRING_APPLICATION_JSON: '{
+        "hawkbit.artifact.url.protocols.download-http.rel": "download-http",
+        "hawkbit.artifact.url.protocols.download-http.hostname": "hawkbit.dev.example",
+        "hawkbit.artifact.url.protocols.download-http.protocol": "https",
+        "hawkbit.artifact.url.protocols.download-http.supports": "DMF,DDI",
+        "hawkbit.artifact.url.protocols.download-http.ref": "{protocol}://{hostnameRequest}/{tenant}/controller/v1/{controllerId}/softwaremodules/{softwareModuleId}/artifacts/{artifactFileName}"
+      }'
+    restart: no
+    ports:
+      - "8080:8080"
+    labels:
+      NAME: "hawkbit"
+
+```
+You need to add the configurations that start with the hawkbit.artifact.url.protocols to the either The link is generated by Hawkbit and then transmitted to the client, enabling the client to download the firmware package through this link. Remember to replace "hostname" with your actual hostname.
+
+In Hawkbit's UI section, under system configuration, make sure to select Allow targets to authenticate via a certificate authenticated by a reverse proxy and input the issuer hash as "Hawkbit". This can be whetever you have configured in the nginx configuration in `proxy_set_header X-Ssl-Issuer-Hash-1`
+
+![Reverse Proxy Settings](../../images/security/reverseProxySettings.png)
+
+After successfully generating your certificates with the correct chain, deploying your Nginx and Hawkbit services with appropriate configurations, and updating the settings on the device side, you will be able to establish a certificate-based authentication mechanism. This will eliminate the necessity of sharing a security token with the server.
+
+#### Testing
+
+You can test the communication by using the Curl command below to see if you successfully implemented Mutual TLS:
+
+`curl -L -v --cert Client.crt --key Client.key --cacert CA2.pem https://hawkbit.dev.example.com/default/controller/v1/{device-id}`
+
+In the UI, after uploading an SWU package and requesting a firmware update, you can use the link below to attempt to install the software package.
+
+`curl -L -v --cert Client.crt --key Client.key --cacert CA2.pem https://hawkbit.dev.example.com/default/controller/v1/{device-id}/softwaremodules/{artifact-id}/artifacts/hawkbit_updated_5.swu --output outputfile`
+
+#### References
```

Comment:
I think references shall be removed here. 
They may disappear, become outdated and so on. 
Users are free to browse for more info by themselfs

Reply:
Removed